externals/llvm-external-projects/torch-mlir-dialects/include/torch-mlir-dialects/Dialect/Tcp/IR/TcpOps.td

```diff
@@ -191,4 +191,92 @@ def Tcp_BroadcastOp : Tcp_Op<"broadcast", [
   let hasVerifier = 1;
 }
 
+def Tcp_YieldOp : Tcp_Op<"yield", [Terminator, Pure]> {
+  let summary = "Yields values from regions in Tcp";
+
+  let description = [{
+    Terminates and yields values from regions in Tcp.
+
+    The values that are yielded should correspond to the parent operation's
+    results.
+  }];
+
+  let arguments = (ins
+    Variadic<Tcp_TensorOrScalar>:$ins
+  );
+
+  let builders = [
+    OpBuilder<(ins), [{ }]>,
```

Comment:
If you include InferReturnType.td (or whatever it is called) do these get generated?

Reply:
I tried including a few different `.td` files and none of them helped generate this. 

@sjarus Do you know what to include for this?

Reply:
I assume @jpienaar means the inferReturnTypeComponents interface. You can check out include/mlir/Dialect/Tosa/IR/TosaOps.td for usage of that.

Reply:
Ah this one I figured as variadic should allow empty there.